events/world-moveit-day-2018/index.markdown

```diff
@@ -39,7 +39,7 @@ World MoveIt! Day is an international hackathon to improve the MoveIt! code base
   - Organizer: Ryosuke Tajima
 - [Southwest Research Institute](https://www.swri.org/)/[ ROS-Industrial Consortium Americas](https://rosindustrial.org/ric-americas/), San Antonio, USA
   - Organizer: Matt Robinson
-  - [Event Details](https://www.evite.com/event/030AUF672VIUQMVHQEPI2KHJ4AIZNA/rsvp?utm_source=NA&utm_medium=sharable_invite&utm_campaign=send_sharable_link)
+  - [Event Details](https://www.evite.com/event/030AUF672VIUQMVHQEPI2KHJ4AIZNA/)
```

Comment:
The link gets rerouted to https://www.evite.com/event/030AUF672VIUQMVHQEPI2KHJ4AIZNA/activity

Reply:
Somehow the same comment got repeated